src/src/models/CheckboxModel.ts

```diff
@@ -0,0 +1,15 @@
+import OptionModel from "./OptionModel";
+
+interface CheckboxModel {
+    customClass: String;
+    fieldIcon: String;
+    fieldGroupName: String;
+    disabled: Boolean;
+    suffix: String;
+    prefix: String;
+    fieldName: String;
+    fieldValue: String;
```

Comment:
Codacy has a fix for the issue: [Don't use `String` as a type. Use string instead](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
    fieldValue: string;
```